polkadot-parachains/src/service.rs

```diff
@@ -51,12 +53,8 @@ use sp_runtime::{
 use std::sync::Arc;
 use substrate_prometheus_endpoint::Registry;
 
-pub use sc_executor::NativeExecutor;
```

Comment:
Why did you moved this?

Reply:
I guess it is analyzer sorting the imports.